rfcs/0024-incremental-ledger-catchup.md

```diff
@@ -0,0 +1,81 @@
+## Terminology
```

Comment:
This should be moved somewhere later in the document

Reply:
I put it here so that people won't be confused by what "history_root" is. I guess I invented this word.

Reply:
Sure ... but it would be nice to keep the usual structure, with `Summary` first. You could include the definition in the Summary.